angular/test/test-app/src/app/modal-example/modal-example.component.ts

```diff
@@ -16,6 +16,8 @@ export class ModalExampleComponent implements OnInit, ViewWillLeave, ViewDidEnte
   willLeave = 0;
   didLeave = 0;
 
+  modal: HTMLElement;
```

Comment:
Is this added inside of the component rendered inside of the modal, or the component that renders the modal itself?

Reply:
This property is assigned to the component class, similar to how we pass component props through with controller modals. 

Adding this property manually is just to allow Typescript to understand that member exists. Technically it's always been there/available on the class instance, just wasn't typed.